### DISCUSSION TITLE
How to use the /docs feature in the Assistant tab?

### DISCUSSION THREAD

**Discussion by JohnnyHoff** [zed-industries/zed#15416] [Feature Requests]
I couldnt find any docs about the docs feature (the irony!)
How does it work? I would love to be able to quickly add some docs to a LLM when it got issues with a task

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
